src/main/java/codeu/model/data/User.java

```diff
@@ -90,4 +90,24 @@ public Instant getCreationTime() {
   public void adminify() { this.isAdmin = true; } // maybe rename to promote()
 
   public void unadminify() { this.isAdmin = false; } // maybe rename to demote()
+
+  /** Override equality, so users are compared based on their ID */
+  @Override
+  public boolean equals(Object o) {
+
```

Comment:
Ditto here (for whitespace and redundant logic)

Reply:
Done.